ens/async_ens.py

```diff
@@ -1,3 +1,4 @@
+# flake8: noqa: B008
```

Comment:
I don't think we should ignore whole files like this unless we have good reason to. We can resolve the three specific issues by setting arguments to `None` and addressing them in the method with `if x is None: {{set the default value}}`.

This guarantees it's not just set once. Even if that's intended behavior, we should add `noqa: B008` to single lines and explain why they're there with a comment instead of whole files. That would make it easy to mask future errors since it's sneakily up at the top of the file.

Specifically, in this file, we can address:

- `provider` in `__init__()`
- `address` in `setup_address()`
- `new_owner` in `setup_owner()`.

Reply:
I put those there instead of inline because I made a note "talk to Felipe about the B008 ignores". And then forgot about it.